addons/wowl/static/src/components/navbar/navbar.xml

```diff
@@ -14,23 +14,19 @@
     </nav>
   </header>
 
-<ul t-name="wowl.AppsMenu" class="o_menu_apps" owl="1">
-    <li class="dropdown">
-        <a class="full" data-toggle="dropdown" data-display="static" href="#" t-on-click="toggleDropdownMenu">
-            <i class="fa fa-th-large"/>
-        </a>
-        <div class="dropdown-menu" role="menu" t-att-class="{ show: state.showDropdownMenu }">
-            <t t-foreach="menuRepo.getApps()" t-as="app" t-key="app.id">
-                <a role="menuitem" href="#" class="dropdown-item o_app"
-                    t-att-data-menu-id="app.id"
-                    t-att-data-menu-xmlid="app.xmlid"
-                    t-att-data-action-id="app.actionID"
-                    t-on-click="_onMenuClicked(app)">
-                        <t t-esc="app.name"/>
-                </a>
-            </t>
-        </div>
-    </li>
-</ul>
+  <Dropdown t-name="wowl.NavBar.AppsMenu" class="o_navbar_apps_menu" menuClass="'show dropdown-menu'" t-on-dropdown-item-selected.stop="_onMenuClicked" owl="1">
+    <i class="fa fa-th-large"/>
+    <t t-set-slot="dropdown">
 
-</templates>
\ No newline at end of file
+        <t t-foreach="menuRepo.getApps()" t-as="app" t-key="app.id">
+          <DropdownItem value="app" role="menuitem" class="dropdown-item o_app"
+                  t-att-data-menu-id="app.id"
+                  t-att-data-menu-xmlid="app.xmlid"
+                  t-att-data-action-id="app.actionID">
```

Comment:
Do you know what uses those attributes ? Maybe they won't be required in the new web client.

Reply:
For now, I dunno